docs/DeveloperGuide.md

```diff
@@ -256,26 +256,39 @@ Priorities: High (must have) - `* * *`, Medium (nice to have)  - `* *`,  Low (un
-`* *` | user | hide [private contact details](#private-contact-detail) by default | minimize chance of someone else seeing them by accident
-`*` | user with many persons in the address book | sort persons by name | locate a person easily
+`* * *` | user | add a new task or event with a given deadline or date and priority | remember what tasks I have to do in the future. 
+`* * *` | user | add a floating task with no given deadline or date | work on long term tasks that have no given deadline.
+`* * *` | user | mark a task or event as completed | refer to which tasks I have finished or not finished
+`* * *` | user | find a task or event by name | locate details of tasks/events without having to go through the entire list
+`* * *` | user | list upcoming tasks in sorted order | determine which tasks I should do next
+`* * *` | user | edit tasks | update tasks if things change.
+`* * *` | user | undo recent commands | revert tasks/events created by mistake.
+`* * *` | user | receive reminders for upcoming tasks | don't forget about tasks
+`* *` | user | add tasks through plain English | type more naturally than having to write in commands and flags.
+`* *` | user | view tasks in a calendar format | figure out what events/tasks I have upcoming more easily.
+`* *` | user | specify the location to place the data storage | do things with it, like sync it to my Dropbox.
+`* *` | user | bring up the program with a hotkey | pull it up quickly and conveniently when I need to add a task.
+`* *` | user | schedule multiple time blocks with one task | schedule tasks or events that have multiple and different deadlines or dates.
+`* *` | user | auto-complete certain tasks | save time instead of having to type out the whole command.
+`*` | user | manipulate the schedule with a mouse | save time manipulating the calendar directly as opposed to having to go back to the CLI and type in more command rescheduling events (e.g. drag-drop vs. retyping event details)
+`*` | user | view list or calendar in a pop out window | look at it more easily while working on other items.
+`*` | user | have a nice GUI | have an easier time manipulating the information and let the program be more pleasing to my eyes.
 
 {More to be added}
 
 ## Appendix B : Use Cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `JYM` and the **Actor** is the `user`, unless specified otherwise)
 
-#### Use case: Delete person
+#### Use case: Mark Task completed.
 
 **MSS**
 
-1. User requests to list persons
-2. AddressBook shows a list of persons
-3. User requests to delete a specific person in the list
-4. AddressBook deletes the person <br>
+1. User requests to view list of active tasks
+2. System shows list of tasks
+3. User requests to mark a task as completed from the list.
+4. System marks task as completed
+5. System removes task from list of active tasks. <br>
```

Comment:
This can be merged with `4.`. User stories are usually of form user-system-user-system etc., to denote the interactions.